[CFE-715] Enable Cert-Manager Metrics (#92)
openshift/cert-manager-operator: OPERAND_METRICS.md

## Enabling metrics and monitoring for `cert-manager`

Cert-Manager exposes controller metrics in the format expected by [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator). 

> ServiceMonitor resource needs to be created to scrape metrics from cert-manager operand, make sure that your instance of the Prometheus Operator is properly configured to find it.

Reviewer: ```suggestion
ServiceMonitor resource needs to be created to scrape metrics from cert-manager operand, make sure Prometheus Operator is configured with required selectors.
```

> `.spec.serviceMonitorNamespaceSelector` and `.spec.serviceMonitorSelector` fields of prometheus resource should contain corresponding .

Reviewer: actual values need to be mentioned after corresponding
